packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -191,6 +192,9 @@ class BottomNavigationBar extends StatefulWidget {
   /// See [BottomNavigationBarItem.icon] for more information.
   final double iconSize;
 
+  /// Expose heightFactor in order to control the spaces between icon and text.
+  final double heightFactor;
```

Comment:
Is this necessary to support a label-less bottom nav?
 

This comment should describe what the value does, the defaults, and what changing it means. If I'm looking to adjust something on the bottom nav, how do I find it here?

Reply:
I've removed this. I needed this when I had to restrict child widget to `Text` because I need to but extra space between the `icon` and `label`. Now, this is reverted so I no more need this to be exposed.